samples/iot_c_utility/iot_c_utility.c

```diff
@@ -53,7 +53,7 @@ static void show_sastoken_example()
 
 static void show_platform_info()
 {
-    STRING_HANDLE platform_info = platform_get_platform_info();
+    STRING_HANDLE platform_info = platform_get_platform_info(PLATFORM_OPTION_RETRIEVE_SQM);
```

Comment:
What is the reason to expose this option to the user?
Can it be just assumed that if the application is running on Windows it will always try to get the SQM info?

Reply:
There's a requirement for the SQM ID to not be included when using HTTP in the Azure C SDK to limit the size of the string in the request.  It can be assumed that if the app is running on Windows it'll always try to get the SQM ID, just not when the transport protocol is HTTP.  Exposing this option will also allow us to more easily modify the contents of the returned platform info in the future if need be.